### PR TITLE
fix grammar in pkg module

### PR DESCRIPTION
**Pull request by xujingjing-cmss**
Signed-off-by: xujingjing xujingjing_yewu@cmss.chinamobile.com

What type of PR is this?
/kind cleanup

What this PR does / why we need it:
fix spelling error and remove redundant code.

Which issue(s) this PR fixes:
Fixes #

Special notes for your reviewer:

Does this PR introduce a user-facing change?:

### CODE REVIEW

File: pkg/util/flag/flags.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ func (m *ConfigValue) Set(s string) error {
 }
 
 func (m *ConfigValue) String() string {
-	return fmt.Sprintf("%v", bool(*m == ConfigTrue))
```
Comment (fisherxu) [inline pkg/util/flag/flags.go:55]: This file is from K8s repo, let's keep it as above :) Ref: https://github.com/kubernetes/kubernetes/blob/master/pkg/util/flag/flags.go

File: pkg/version/verflag/verflag.go [hunk 2]
```diff
@@ -64,7 +64,7 @@ func (v *VersionValue) String() string {
 	if *v == VersionRaw {
 		return strRawVersion
 	}
-	return fmt.Sprintf("%v", bool(*v == VersionTrue))
```
Comment (fisherxu) [inline pkg/version/verflag/verflag.go:67]: Ditto.

File: pkg/util/flag/flags.go (hunk 1 above)
Comment (xujingjing-cmss) [reply pkg/util/flag/flags.go]: done

File: pkg/version/verflag/verflag.go (hunk 2 above)
Comment (xujingjing-cmss) [reply pkg/version/verflag/verflag.go]: done

**Pull request closed without merging**